docs/develop/01_sdk/05_troubleshoot.md

```diff
@@ -5,6 +5,12 @@ title: Troubleshoot
 
 Solutions and workarounds for common or unresolved issues.
 
+## Node version
+Kilt currently supports NodeJS 14,16, and 18
+
+## Vite version
+Vite is known to work with version ^4.1.0
```

Comment:
What does it mean? With version `^4.1.0` of what?

Reply:
When we were debugging - 1gn0r4nd's 'vite' version was incompatible. So, upgrading from version ~'3.1.0' to the latest `Vite` version helped.